datahub/detector/dtree/config_info.py

```diff
@@ -0,0 +1,7 @@
+class ConfigInfo(object):
+    def __init__(self, conf_file_path):
+        self.load_config(conf_file_path)
+
+    def load_config(self, conf_file_path):
+        """Load config from config file
```

Comment:
We have a centralized package for datahub config, please move all your config options to datahub/conf and follow the conventions.